geozero/src/csv/csv_error.rs

```diff
@@ -0,0 +1,14 @@
+//! CSV Error type.
+use thiserror::Error;
```

Comment:
hmmmm I am slightly confused since isn't that the case for `MvtError` as well?

Reply:
Please move the doc to the enum. It's not a big deal, but no sense in doubling down on "wrong" just for consistency.